DuckDuckGo/CrashCollectionOnboardingView.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  CrashCollectionOnboardingView.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import SwiftUI
+import DuckUI
+import DesignResourcesKit
+
+struct CrashCollectionOnboardingView: View {
+
+    @ObservedObject var model: CrashCollectionOnboardingViewModel
+
+    var body: some View {
+        if #available(iOS 16.0, *) {
+            // Using NavigationStack here in order to fix presentation on the iPad, where `Navigation` would use Split Navigation view.
+            NavigationStack {
+                contents
+            }
+            .background(Color(designSystemColor: .backgroundSheets))
+        } else {
+            NavigationView {
+                contents
+            }
+            .background(Color(designSystemColor: .backgroundSheets))
+        }
+    }
+
+    var contents: some View {
+        VStack(spacing: 0) {
+            ScrollView {
+                VStack(spacing: 24) {
+                    Image("Breakage-128")
+
+                    Text(UserText.crashReportDialogTitle)
+                        .daxTitle1()
+                        .foregroundColor(Color(designSystemColor: .textPrimary))
+
+                    Text(UserText.crashReportDialogMessage)
+                        .multilineTextAlignment(.center)
+                        .daxBodyRegular()
+                        .foregroundColor(Color(designSystemColor: .textPrimary))
+
+                    if let reportDetails = model.reportDetails {
+                        VStack(spacing: 4) {
+
+                            reportDetailsButton
+
+                            if model.isReportVisible {
+                                ZStack {
+                                    Rectangle()
+                                        .foregroundColor(Color(designSystemColor: .container))
+                                        .frame(maxWidth: .infinity, maxHeight: .infinity)
+                                        .cornerRadius(4.0)
+
+                                    Text(reportDetails)
+                                        .multilineTextAlignment(.leading)
+                                        .font(.crashReport)
+                                        .foregroundColor(Color(designSystemColor: .textSecondary))
+                                        .padding(24)
+                                }
```

Comment:
I was wondering whether it would make sense to put this in a scroll view which is horizontally scrollable since text wrapping can make it awkward to read, but TBH there's no right approach here – happy to leave this as-is.

Reply:
That was my first idea and implementation, but we agreed with Bryan that it actually shouldn't be scrolled horizontally ([see the discussion](https://app.asana.com/0/0/1206992056177669/1207072176348345/f)).

That said, the scroll view approach makes rendering faster (not fast, but faster) 🤷